pypesto/profile/profile.py

```diff
@@ -63,7 +66,12 @@ def parameter_profile(
     # Handling defaults
     # profiling indices
     if profile_index is None:
-        profile_index = np.ones(problem.dim_full)
+        list(range(problem.dim_full))
+
+    # check if more detailed options were passed
+    if profile_index_advanced is None:
+        profile_index_advanced = np.array([0] * problem.dim_full)
+        profile_index_advanced[profile_index] = 1
```

Comment:
profile_index_advances is currently not used yet, so maybe exclude for the moment? line 94/102 does not seem to fit to the new form of profile_index yet. nbs would need to be updated too.